src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
While you're at it, I think we should stop mentioning R, as we don't want to list every possible software. Giving the terminology they use is enough. Also, I would mention "Dummy coding" in the first (summary) line: even if it's in the type name, I would find it clearer. Same remarks for other contrasts.